### PR TITLE
feat: ✨ add curse word detection

### PR DESCRIPTION
**Pull request by ctnkaan**
## Fixes Issue 

This PR fixes the following issues:

#13

## Changes proposed

Added a array of curse words. Bot scans the messages if any of them contain a word from the array.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 
Here are some tests
image

### CODE REVIEW

File: Commands/restrictedWords.js [hunk 1]
```diff
@@ -38,14 +38,34 @@ const words = [
...
                 message.author.send(warnEmbed);
                 message.delete();
             }
         }
+
+        for (let i = 0; i < curseWords.length; i++) {
+            if(message.content.toLowerCase().includes(curseWords[i])) {
+                message.reply('Please do not use profane language 🚫. If you think I did a mistake dont worry I am still under development, tag Community Manager and report this 🏷️.');
+                message.author.send(warnEmbed);
+                message.delete();
```
Comment (kaiwalyakoparkar) [inline Commands/restrictedWords.js:67]: Should this be there. What do you think?

Comment (ctnkaan) [reply Commands/restrictedWords.js:67]: Hmmm. I've seen that this is included for tech startups so I thought I would include one for curse words too. Maybe we could remove it and make it only send private message. I think that would be better yea.

Comment (kaiwalyakoparkar) [reply Commands/restrictedWords.js:67]: Private and message under their message should  be good

**Comment by ctnkaan** [comment]
@kaiwalyakoparkar Done! Made the curse words messages private.

**Comment by kaiwalyakoparkar** [comment]
@ctnkaan Good job but I meant to say that both ways. As we do in the edTech startup words. (We message in both in DM and in the public chats )

**Comment by ctnkaan** [comment]
@kaiwalyakoparkar done

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing

**Pull request merged**